### PR TITLE
Merge latest master

### PR DESCRIPTION
**Pull request by tephenavies**

### CODE REVIEW

**Comment by rowanwins** [comment]
@reginapramesti can you just quickly check the IE overlay seeing as your on windows :)

**Review by reginapramesti — APPROVED** [review]

**Comment by reginapramesti** [comment]
Probably worth noting here:

> the CI doesn't load on IE at all, but in saying that neither does the next branch :grimacing:

File: lib/ReactViews/StandardUserInterface/StandardUserInterface.jsx [hunk 1]
```diff
@@ -257,6 +258,7 @@ const StandardUserInterface = observer(
             viewState={this.props.viewState}
           />
           <div className={Styles.storyWrapper}>
+            <InternetExplorerOverlay viewState={this.props.viewState} />
```
Comment (soyarsauce) [inline lib/ReactViews/StandardUserInterface/StandardUserInterface.jsx:261]: Simply don't mount this at all for next

Comment (reginapramesti) [reply lib/ReactViews/StandardUserInterface/StandardUserInterface.jsx:261]: Oops I already merged this, I'll create a one-line PR that removes this line from `next`

**Pull request merged**